benchmarks/experimental/benchmark_dataset.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
```

Comment:
Missing license section?

Reply:
I think there's a discrepancy across many scripts. I noticed that this is the comment that is present in most of the other scripts (see, benchmarks/pipe.py, etc). However, there's an extra license section in benchmarks/experimental/offload.py 

Let me make an issue and we can address it separately.